### PR TITLE
V 0 3 0 unit tests

### PR DESCRIPTION
**Pull request by shannonlal**
This pull request is for adding some basic unit tests.  I found it was helpful for me as a new developer to the project to put some basic unit testing in place.  I hope this will help other developers moving forward  and make it easier for other people to contribute to the project.  I know the unit tests are very simple and are overkill so I welcome comments from other developers.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,70 +1,70 @@
 var inq = require('inquirer'),
-	es  = require('event-stream');
+es  = require('event-stream');
```
Comment (Freyskeyd) [inline index.js]: The indentation format is a bit weird I think^^

File: index.js [hunk 2]
```diff
@@ -1,70 +1,70 @@
...
-	},
-	
-	inq: inq
-};
+		prompted = true;
+	});
+},
+
+inq: inq
+};
```
Comment (Freyskeyd) [inline index.js]: You need a breakline here

File: test/confirm-spec.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+var proxyrequire = require('proxyquire');
+/**
+ * The following test spec will verify the operation of the confirm function
+ */
+describe('gulp confirm function unit tests', function() {
+  describe('verify that confirm function operates correctly', function() {
+    it('verify that confirm converts string message to options', function( done ){
+        var prompt = function( listOptions ){
+            return new Promise( (resolve,reject)=>{
+                //
```
Comment (Freyskeyd) [inline test/confirm-spec.js]: Unneeded

File: test/confirm-spec.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+/**
+ * The following test spec will verify the operation of the confirm function
+ */
+describe('gulp confirm function unit tests', function() {
+  describe('verify that confirm function operates correctly', function() {
+    it('verify that confirm converts string message to options', function( done ){
+        var prompt = function( listOptions ){
+            return new Promise( (resolve,reject)=>{
+                // 
+                if( ( Array.isArray( listOptions) ) && (typeof listOptions[0] !== 'string') ){
```
Comment (Freyskeyd) [inline test/confirm-spec.js]: Do you use eslint as a style checker?
```javascript

if (Array.isArray(listOptions) && typeof listOptions[0] !== 'string') {
   ... 
} else {
   ...
}

File: test/confirm-spec.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+var proxyrequire = require('proxyquire');
+/**
+ * The following test spec will verify the operation of the confirm function
+ */
+describe('gulp confirm function unit tests', function() {
+  describe('verify that confirm function operates correctly', function() {
+    it('verify that confirm converts string message to options', function( done ){
+        var prompt = function( listOptions ){
+            return new Promise( (resolve,reject)=>{
```
Comment (Freyskeyd) [inline test/confirm-spec.js]: you need space before and after `=>`

**Comment by shannonlal** [comment]
Ok.  Thanks for comments.  Adding in eslint to the project now and getting it up and running.  Just running into a parse error issue.  Once it is resolved I will check it in

**Pull request merged**